features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -456,6 +456,34 @@
 				<AttributeID>oneTimePassword</AttributeID>
 				<Description>One Time Password</Description>
 		    </Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/extendedRef</ClaimURI>
+				<DisplayName>Extended Ref</DisplayName>
+				<!-- Proper attribute Id in your user store must be configured for this -->
+				<AttributeID>extendedRef</AttributeID>
+				<Description>Extended Ref</Description>
+			</Claim>
+			<Claim>
+				<ClaimURI>http://wso2.org/claims/extendedDisplayName</ClaimURI>
+				<DisplayName>Extended Display Name</DisplayName>
+				<!-- Proper attribute Id in your user store must be configured for this -->
+				<AttributeID>extendedDisplayName</AttributeID>
```

Comment:
Did you add these custom attributes to embedded ldap schema?

Reply:
Yes. PR can be found in https://github.com/wso2-extensions/identity-userstore-ldap/pull/16

Reply:
Fixed in https://github.com/wso2-extensions/identity-userstore-ldap/pull/16